frankenstein/distributor.go

```diff
@@ -0,0 +1,102 @@
+package frankenstein
+
+import (
+	"sync"
+
+	"github.com/prometheus/prometheus/storage"
```

Comment:
We usually sort import groups like this in Prometheus:

- stdlib packages
- repo-external packages
- repo-internal packages

So just switch the last two.